pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "graphene_pydantic"
-version = "0.6.0"
+version = "0.6.1"
```

Comment:
@necaris any opinions on the version bump being in the same or separate PR? Happy to comply.

Reply:
Same PR seems fine to me.